client/src/pages/MoviesList/utility.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react'
+import { useConfiguration } from '../../providers/Configuration'
+import { authFetch } from '../../auth'
+
+export const useTurns = () => {
```

Comment:
since this file just contains a hook you can rename the file to`useTurns` and put it in the `hooks` folder. `src/hooks`